lib/pytap13.py

```diff
@@ -176,7 +177,7 @@ def _parse(self, source):
             self.tests.append(Test('not ok', len(self.tests), comment=comment))
 
     def parse(self, source):
-        if isinstance(source, (str, unicode)):
+        if isinstance(source, six.string_types):
```

Comment:
It should not be part of the 'python3: using six to iteritems()' commit, because it is not about iterating over a dictionary.

Reply:
replaced to `string_types` imported from `lib/utils.py`